lib/viewer.coffee

```diff
@@ -112,15 +113,16 @@ class Viewer extends Disposable
     return true
 
 class PDFView
-  constructor: (url) ->
+  constructor: (url,tabTitle) ->
```

Comment:
To avoid confusion, please change the variable name to `title` as this view is also used for the viewer window.